app/stacks/uk-west/back-office/storage-account.tf

```diff
@@ -102,3 +102,19 @@ resource "azurerm_storage_account" "function_storage" {
 
   tags = local.tags
 }
+
+module "apply_blob_container_legal_hold" {
+  blob_store_account_container_pairs = [
+    [
+      azurerm_storage_account.back_office_documents,
+      azurerm_storage_container.back_office_published_documents_container

```

Comment:
We want to apply this to the internal one: `back_office_document_service_uploads_container` I believe.